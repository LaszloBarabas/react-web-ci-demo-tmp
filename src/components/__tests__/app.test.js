
//import React from 'react';
//import App  from '../app';
////import renderer from 'react-test-renderer';
//import ShallowRenderer from 'react-test-renderer/shallow';
//
//it('App renders correctly', () => {
//    const renderer = new ShallowRenderer(); 
//
//    const appcomponent = renderer.create(<App />).toJSON(); 
//    expect(appcomponent).toMatchSnapshot(); 
//}); 






// we will use jest and enzyme 
//


import React from 'react';
import {shallow, mount, configure} from 'enzyme'; 
import Adapter from 'enzyme-adapter-react-16'; 

import App  from '../app'; 

configure({adapter: new Adapter()}); 

describe('App component  ', () => {
    it ('should render correctly ', () => {
        var webcomponent = shallow( <App /> ); 
        expect(webcomponent).toMatchSnapshot(); 

    }); 

} ); 
