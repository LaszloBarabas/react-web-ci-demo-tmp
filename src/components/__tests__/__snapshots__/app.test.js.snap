// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component   should render correctly  1`] = `
<div>
  React simple starter
  <div>
    <hr />
    <Connect(CitiesContainer) />
    <Connect(SelectedCityContainer) />
    <hr />
  </div>
  <div>
    <hr />
    <Connect(SearchContainer) />
    <hr />
    <Connect(TemperatureInTheLocationContainer) />
    <hr />
  </div>
</div>
`;
